mitosheet/src/components/taskpanes/Graph/GraphSidebar.tsx

```diff
@@ -232,6 +237,18 @@ const GraphSidebar = (props: {
         changeLoadingGraph(getGraphAsync);
     }
 
+    // Toggles the safety filter component of the graph params
+    const toggleSafetyFilter = (): void => {
+        const newSafetyFilter = !graphParams.safetyFilter
+
+        console.log(newSafetyFilter)
+
+        setGraphParams({
+            ...graphParams,
+            safetyFilter: newSafetyFilter
+        })
```

Comment:
Do the toggling inside the call back for setGraphParams, if you don't mind. It should be a `React.Dispatch<React.SetState>>` or whatever the hell if it isn't already.

Reply:
I vote we leave it for now, because the entire graphSidebar.tsx file sets parameters using the `setGraphParams` function, which is responsible for updating the parameters and updating the saved parameters. 

If we're not happy with this approach, lets make the move completely when we redo all of this code.